Sources/CALayer.swift

```diff
@@ -133,7 +133,7 @@ open class CALayer {
             guard newBounds != bounds else { return }
             onWillSet(keyPath: .bounds)
 
-            if bounds.size != newBounds.size {
+            if !isPresentationForAnotherLayer && bounds.size != newBounds.size {
```

Comment:
how is this related to `contentOffset` @michaelknoch? That's the bit I don't understand right now

Reply:
on iOS this Check does not exist. They seem to always relayout the superlayer when bounds changes. 

But there is this exception where they do not relayout when setting the content offset of a scrollview.

This surprised me because from our understanding setting the contentOffset directly mutates the bounds.origin.


What does this mean? Initially
I thought we should remove the check and always relayout when bounds change (not only when bounds.size changes) but then we see this massive amount of relayouts when we scroll in our scrollview. Does this make sense @ephemer ? It's pretty hard to explain, it will be way easier to demonstrate.

Reply:
ahhhh ok sorry I didn't get the point about iOS also laying out the superview when `.origin` changes as well